content/S2016/S16-event-square.md

```diff
@@ -0,0 +1,27 @@
+title: Square Tech Workshop: Becoming Ok with Okhttp
+Date: 2016-05-18
+Category: Events
+Tags: Square, workshop, open source, Okhttp
+Slug: square-event-s16
+Author: Bhargavi Dameracharla
+Summary: Technical workshop on open source HTTP & HTTP/2 client
+
+Square is holding a technical workshop on their open source HTTP & HTTP/2
+client for Android and Java applications.
+Join us to learn more about Okhttp and get a chance to speak with their
+engineers!
+Food will be available, including vegetarian, vegan, and gluten-free
+options.
+
+There is an attendance cap of 70 people (50 undergrads and 20 grads),
+so make sure to [register]
+(https://www.eventbrite.com/e/square-tech-workshop-becoming-ok-with-okhttp-tickets-25360009471)
```

Comment:
oh oops - you can't break up links like that
see https://wics-site-pr-210.herokuapp.com/